doc/release-notes.md

```diff
@@ -0,0 +1,5 @@
+# Orchestrate Convert API Release Notes
```

Comment:
Remove this file in favor of the GitHub releases functionality.

Reply:
Done